scripts/variantstore/docs/aou/AOU_DELIVERABLES.md

```diff
@@ -80,6 +80,7 @@
     - This workflow transforms the data in the vet tables into a schema optimized for callset stats creation and for calculating sensitivity and precision.
     - It will need to be run twice. Once with `only_output_vet_tables` set to "true" (the default value is `false`) and once with `control_samples` set to "true" (the default value is `false`).
     - See [naming conventions doc](https://docs.google.com/document/d/1pNtuv7uDoiOFPbwe4zx5sAGH7MyxwKqXkyrpNmBxeow) for guidance on what to use for `extract_table_prefix` or cohort prefix, which you will need to keep track of for the callset stats. Use the same prefix for both runs.
+    - **NOTE** Set `use_compressed_references` to true.
```

Comment:
We should consider having this workflow auto-detected that the references are compressed

Reply:
Totally agree, created VS-1138.